webpack.config.js

```diff
@@ -34,12 +35,15 @@ module.exports = {
       {
         test: /\.scss$/,
         use: [
-          'style-loader',
+          {
+            loader: 'style-loader',
+            options: {attributes: {id: `${packageData.name}`}}
+          },
           {
             loader: 'css-loader',
             options: {
               modules: {
-                localIdentName: '[name]__[local]___[hash:base64:5]',
+                localIdentName: `${CSS_MODULE_PREFIX}-ap-[local]`,
```

Comment:
why playkit ?
also, if you want to make this a cross player standard (and also use it in that PR we have in plugin-example), lets discuss it with the team